package.json

```diff
@@ -52,5 +53,8 @@
     "ignore": [
       "soljson.js"
     ]
+  },
+  "solc": {
+    "sha256": "a52d6dddca6c3df5a6364cc11f93a6dd1c58e38e8c740f27e39f98e43cdaafce"
```

Comment:
Alternatively this could come from solc-bin (there was a patch for it ages ago).